### PR TITLE
adjust public WPS outputs sub-dir for weaver

### PR DESCRIPTION
**Pull request by fmigneault**
## Overview

Adjust public WPS outputs sub-dir for Weaver and Cowbird to align.

## Changes

**Non-breaking changes**

- Fix `weaver` and `cowbird` inconsistencies for `public` WPS outputs directory handling.

  Because `cowbird` needs to mount multiple directories within the user-workspace for `jupyterhub`, it needs to define
  a dedicated `public/wps_outputs` sub-directory to distinguish it from other `public` files not part of WPS outputs.
  However, for WPS birds, other files than WPS outputs are irrelevant, and are therefore mounted directly in their
  container. The variable `PUBLIC_WORKSPACE_WPS_OUTPUTS_SUBDIR` was being misused in the context of `weaver`,
  causing WPS output URLs for `public` context to be nested as `/wpsoutputs/weaver/public/wps_outputs/{jobID}`
  instead of the intended location `/wpsoutputs/weaver/public/{jobID}`, in contrast to user-context WPS outputs
  located under `/wpsoutputs/weaver/users/{userID}/{jobID}`.

**Breaking changes**
- n/a

## Related Issue / Discussion

- [x] Depends on https://github.com/Ouranosinc/pavics-sdi/pull/314 for CI tests to succeed.

birdhouse_daccs_configs_branch: master
birdhouse_skip_ci: false

### CODE REVIEW

File: birdhouse/components/cowbird/default.env [hunk 1]
```diff
@@ -62,6 +70,7 @@ COWBIRD_MONGODB_DATA_DIR='${DATA_PERSIST_ROOT}/mongodb_cowbird_persist'
 DELAYED_EVAL="
   $DELAYED_EVAL
   COWBIRD_MONGODB_DATA_DIR
+  COWBIRD_PUBLIC_WORKSPACE_WPS_OUTPUTS_SUBDIR
```
Comment (fmigneault) [inline birdhouse/components/cowbird/default.env:73]: @mishaschwartz 
I do not recall if I need to had `COWBIRD_PUBLIC_WORKSPACE_WPS_OUTPUTS_SUBDIR` to `EXTRA_VARS` or not when it is already in `DELAYED_EVAL`. Is it included automatically?

Comment (mishaschwartz) [reply birdhouse/components/cowbird/default.env:73]: No it isn't. You should add it EXTRA_VARS as well

**Review by mishaschwartz — APPROVED** [review]
Looks good to me, pending adding the variable to `EXTRA_VARS`

**Review by ChaamC — APPROVED** [review]

**Comment by fmigneault** [comment]
Running http://daccs-jenkins.crim.ca/job/DACCS-iac-birdhouse/2229/ with combined https://github.com/bird-house/birdhouse-deploy/tree/fix-weaver-public-wps-outputs and https://github.com/Ouranosinc/pavics-sdi/tree/fix-weaver-nested-output-id.

**Pull request merged**